test/Tests/src/Semantic/Warnings_Spec.enso

```diff
@@ -0,0 +1,54 @@
+from Standard.Base import all
+
+polyglot java import java.lang.Long
+
+import Standard.Test
+
+get_warnings val = Vector.Vector (Warning.get_all val)
+
+type My_Warning reason
+
+type My_Type a b c
+My_Type.my_method = this.a + this.b + this.c
+
+spec = Test.group "Dataflow Warnings" <|
+    Test.specify "should allow to attach multiple warnings and read them back" <|
+        x = 1233
+        y = Warning.attach x "don't do this"
+        z = Warning.attach y "I'm serious"
+        here.get_warnings z . should_equal ["I'm serious", "don't do this"]
```

Comment:
Shouldn't `attach` at the new warnings at the back instead of at the front?

Makes sense if we want warnings to go LIFO, but then it seems inconsistent with how in the expression: `a+b` technically `a` is computed first, but its warning also goes first.

Reply:
Newest warning goes first. Though, I'll probably end up having a logical clock for these, when I actually implement the warnings functionality. This is just to have a notion of tainting values with stuff, the user-facing part is to be implemented.